scripts/combine_lightcurves.py

```diff
@@ -77,20 +74,15 @@ def upload_output(self, filename, data):
         :param filename: the name of the file to upload
         :param data: the data to upload
         """
-        local_path = '{}/{}'.format(self.temp_dir, filename)
-        os.makedirs(os.path.dirname(local_path), exist_ok=True)
-        with open(local_path, 'w') as fo:
-            json.dump(data, fo)
-
-        pan_storage = self.storage
-        pan_storage.upload(local_path, remote_path=filename)
+        storage = self.storage
+        storage.upload_string(json.dumps(data), filename)
```

Comment:
Should probably check for an error here on the json encoding. For the most part we know what will be going into `data` but you never know.